src/service/constants.js

```diff
@@ -1 +1,3 @@
-export const API_URL = process.env.REACT_APP_API_URL;
+export const API_URL = 'http://localhost:4000';
```

Comment:
this is def going to break when deploying.

Reply:
fixed. now using process.env again